mos/settings/common.py

```diff
@@ -143,3 +144,9 @@
 HOS_INTRODUCTION = True
 HOS_PROJECTS = True
 HOS_RECENT_CHANGES = True
+
+# ----------------- Jour Fixe Reminder ------------
+MOS_JF_DAYS_IN_ADVANCE = 3
+MOS_JF_DB_ID = 2 # id of events of type "Jour Fixe" in the database
```

Comment:
This is the only thing I feel unsure about. The ID can probably be queried from the database somehow, but that would still leave the string "Jour Fixe" or something like that. I think it's ok for now.